src/OpenTelemetry.Extensions/README.md

```diff
@@ -84,3 +84,27 @@ var tracerProvider = Sdk.CreateTracerProviderBuilder()
 Warning: The baggage key predicate is executed for every baggage entry for each
 started activity.
 Do not use slow or intensive operations.
+
+### RateLimitingSampler
+
+The rate limiting sampler is a sampler that will limit the number of traces to
+the specified rate per second. It is typically used in conjunction with the ParentBasedSampler
+to ensure that the rate limiting sampler is only applied to the root spans. If
+a request comes in without a sampling decision, the rate limiting sampler will
```

Comment:
```suggestion
an `Activity` creation request comes in without a sampling decision, the rate limiting sampler will
```

Reply:
this statements feels incorrect. RateLimitingSampler does not check whether there is a parent decision or not. that is upto end user to ensure by wrapping this inside ParentBased.